scripts/src/main/resources/scripts/functions

```diff
@@ -1291,12 +1304,15 @@ function doInstall() {
     doFail "Missing software argument for doInstall: ${*}"
   fi
   local result=0
-  if [ "${url/.git/}" != "${url}" ]
+  if [ "${url/.git/}" != "${url}" ] && [ "$isOnline" = 0 ]
   then
     doGitPullOrClone "${target_path}" "${url}"
     return
   fi
```

Comment:
You introduced a bug here:
if we have a git repo we want to pull or clone it and then return.
In case we are offline, you will now skip the return causing followup errors.